samples/README.md

```diff
@@ -101,3 +101,58 @@ For more information, see https://cloud.google.com/speech/docs
 
 [shell_img]: //gstatic.com/cloudssh/images/open-btn.png
 [shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-speech&page=editor&open_in_editor=samples/README.md
+
+### Speech Recognition betaFeatures
+
+View the [source code][betaFeatures_code].
+
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-speech&page=editor&open_in_editor=samples/betaFeatures.js,samples/README.md)
+
+__Usage:__ `node betaFeatures.js --help`
+
+```
+betaFeatures.js <command>
+
+Commands:
+  betaFeatures.js Diarization                 Isolate distinct speakers in an audio file
+  betaFeatures.js DiarizationGCS              Isolate distinct speakers in an audio file located in a Google Cloud
+                                              Storage bucket.
+  betaFeatures.js multiChannelTranscribe      Differentiates input by audio channel in local audio file.
+  betaFeatures.js multiChannelTranscribeGCS   Differentiates input by audio channe from GCS audio file.
+  betaFeatures.js multiLanguageTranscribe     Transcribes multiple languages from local audio file.
+  betaFeatures.js multiLanguageTranscribeGCS  Transcribes multiple languages from GCS audio file.
+  betaFeatures.js wordLevelConfidence         Detects word level confidence from local audio file.
+  betaFeatures.js wordLevelConfidenceGCS      Detects word level confidence from GCS audio file. 
+
+Options:
+  --version                             Show version number             [boolean]
+  --speechFile, -f                [string] [default: "./resources/commercial_mono.wav"]
+  --gcsUri, -u                    [string] [default: "gs://cloud-samples-tests/speech/commercial_mono.wav"]
+  --speechFileGnome, --gf         [string] [default: "./resources/Google_Gnome.wav"]
+  --gcsUriStereo, --us            [string] [default: "gs://cloud-samples-tests/speech/commercial_stereo.wav"]
+  --multiSpeechFile, --ms         [string] [default: "./resources/multi.wav"]
+  --multiSpeechUri, --msu         [string] [default: "gs://cloud-samples-tests/speech/multi.wav"]
+  --speechFileStereo, --fs        [string] [default: "./samples/resources/commercial_stereo.wav"]
+  --brooklynBridgeSpeechFile, --bb
```

Comment:
Whoa, where did this requirement come from?

Reply:
Ah, nevermind – it was there already!? Craziness

Reply:
needed regen, just pushed!